Documentation/microsoft-git.md

```diff
@@ -0,0 +1,39 @@
+Details about the Microsoft/Git Fork
+====================================
+
+The [`microsoft/git` fork of Git](https://github.com/microsoft/git)
+is used to take early performance patches or custom patches for
+[VFS for Git](https://github.com/microsoft/git).
+
+Integration Testing with C# Layer
+---------------------------------
+
+If you have a pull request in `microsoft/git`, then you
+can test integration in VFS for Git using a pull request
+there.
+
+1. Run the [`git - build installers`](https://dev.azure.com/gvfs/ci/_build?definitionId=11)
+   using "Branch/tag" as `refs/pull/<pr-number>/merge`. Leave "package.preview.extension"
+   as `-pr`.
+
+2. Create a topic branch in VFS for Git and modify the
```

Comment:
Create a topic branch in VFS4G to consume your custom built Git installer.

Modify the ...